src/commands/dxb/profile/passwordpolicies/fix.ts

```diff
@@ -29,26 +27,48 @@ export default class PasswordPoliciesMerge extends SfdxCommand {
     protected orgname:string;
 
     public async run() {
-        let profilename = this.flags.profilename;
-        let sourcepath = this.flags.sourcepath;
-        this.orgname = this.org.getUsername();
-        if (profilename) {
-            try {
-                //do something
-            } catch (err) {
-                console.log(`Could not convert ${profilename}`);
+        const sourcepath = this.flags.sourcepath;
+        try {
+            // check if sourcepath exists and continue
+            fs.accessSync(sourcepath);
+            // retrieve all password policies from target org, these have a different timestamp appended to the file name than the source files
+            fs.mkdirSync('targetOrgPolicies/main/default', { recursive: true });
+            const targetOrgPolicies: { filePath: string, fullName: string }[] = this.getAllPasswordPolicies();
+
+            // get the file names for the source files from the source directory
+            let sourceFiles:string[] = fs.readdirSync(sourcepath);
+            if (sourceFiles.length === 0) {
```

Comment:
don't throw an error if source is empty. Just ignore. Maybe just a warning is good

Reply:
The reason I check this is because I do not want to wrap the next lines between one if-test sourceFiles.length > 0. Maybe log a warning here and return undefined to resolve the async method?